seatunnel-api/src/main/java/org/apache/seatunnel/api/table/factory/FactoryUtil.java

```diff
@@ -321,6 +322,7 @@ public static SeaTunnelTransform<?> createAndPrepareTransform(
         TableFactoryContext context =
                 new TableFactoryContext(
                         Collections.singletonList(catalogTable), options, classLoader);
+        ConfigValidator.of(context.getOptions()).validate(factory.optionRule());
```

Comment:
move to this pr
https://github.com/apache/incubator-seatunnel/pull/4424/

Reply:
done